theme/less/gouvfr-dataset.less

```diff
@@ -210,68 +210,6 @@ body.dataset-display {
         }
     }
 
-    .reuses-list.in-modal {
```

Comment:
Used in `front/dataset/add-reuse-modal.vue`?

Reply:
Gotcha. I have not managed to find a flow which makes me stumble on this use case :-(
Do you have an idea how I could preview it?

Reply:
What about my comment from 7/11 (associate a dataset to a reuse after creating it)? (Don’t quite remember what this is all about since then)

Reply:
Hm I think I tried but I did not have any dataset to add. Will investigate again thank you 👍

Reply:
:+1: